src/Program.cs

```diff
@@ -25,7 +25,12 @@
 
 namespace convert_tool
 {
-  internal class Program
+    static class Globals
+    {
+        public const string stbroot = "../3DDATA/stb/";
```

Comment:
Can we make this a command line parameter? And default it to "./". This way, people can run it from anywhere and we have some sound default.